src/util/apiUtils.js

```diff
@@ -205,15 +230,25 @@ export const parseApiResponse = requestUrl => ([response, body]) => {
  *   `externalRequest` for the query
  */
 export const buildRequestArgs = externalRequestOpts =>
-	({ endpoint, params, flags }) => {
+	({ endpoint, params, flags, meta={} }) => {
 		const dataParams = querystring.stringify(params);
 		const headers = { ...externalRequestOpts.headers };
 		let url = encodeURI(`/${endpoint}`);
 		let body;
 		const jar = createCookieJar(url);
 
-		if (flags) {
-			headers['X-Meetup-Request-Flags'] = flags.join(',');
+		if (flags || meta.flags) {
```

Comment:
just moving `flags` from a top-level Query property to `meta.flags`. All the documentation now advises the latter but for awhile it was defined as a top-level property so this takes care of both formats. I think there's only one flag actually being used in mup-web, so it should be easy to drop the old style completely - I'll open a ticket for that.